garrysmod/lua/includes/extensions/util.lua

```diff
@@ -220,12 +232,12 @@ function util.Timer( startdelay )
 
 	local t = {}
 	setmetatable( t, T )
+	t.m_timestamp = -1.0

```

Comment:
Since the timer starts immediately when the function is called, this should be set to `CurTime()`. Perhaps even simply call `Start( startdelay )` here

Reply:
Changed the util.Timer function to call the Start function upon creation.